bdk-ffi/Cargo.toml

```diff
@@ -18,14 +18,17 @@ path = "uniffi-bindgen.rs"
 default = ["uniffi/cli"]
 
 [dependencies]
-bdk_wallet = { version = "1.0.0-alpha.12", features = ["all-keys", "keys-bip39"] }
-bdk_esplora = { version = "0.14.0", default-features = false, features = ["std", "blocking", "blocking-https-rustls"] }
-bdk_electrum = { version = "0.14.0" }
-bdk_sqlite = { version = "0.1.0" }
-bdk_bitcoind_rpc = { version = "0.10.0" }
+bdk_wallet = { version = "1.0.0-alpha.13", features = ["all-keys", "keys-bip39"] }
+bdk_esplora = { version = "0.15.0", default-features = false, features = ["std", "blocking", "blocking-https-rustls"] }
+# NOTE: This is a temporary workaround to use the electrum-client with the use-rustls-ring feature. It points to a fork
+#       of bdk in which the bdk_electrum library uses the electrum-client with the use-rustls-ring feature.
+bdk_electrum = { git = "https://github.com/thunderbiscuit/bdk/", package = "bdk_electrum", branch = "feature/electrum-client-ring-ffi-alpha13", default-features = false, features = ["use-rustls-ring"] }
```

Comment:
Added a tracking issue for bdk-ffi (that references your open PR's): https://github.com/bitcoindevkit/bdk-ffi/issues/567